packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -862,6 +863,8 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
             `SELECT * FROM ${dataSourceMetadata.schema}."viewField"
           WHERE "viewId" = '${view[0].id}'`,
```

Comment:
I'm surprised we just take the first view randomly here, seems a bit random? I would have assumed the index view was updated if it was just one? (or the first by position?). Again maybe I'm missing context